### DISCUSSION TITLE
[Unstable] The particles emitted from burning affliction gradually out of the correct position when the emitter source or the submarine moves

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14254] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The particles emitted by the burning affliction are very messy.

https://github.com/FakeFishGames/Barotrauma/assets/73634685/09b1461e-2932-4a04-88fd-f032a14d59ab

I think it would be better to bind particles to targets and smoothly follow their positions.
Of course, particles should not completely follow targets, but just adding an offset(from the position where the particle is created to the position of the targets) to their current position, because the rising of the flames also needs to be taken into account.
Not only that, they will also become more messier when the submarine is sinking in high velocity.

### Reproduction steps

1. give burning affliction to yourself
2. run and observe the flames
3. make the submarine sinking severly and keep observing

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this.

![image](https://github.com/user-attachments/assets/0c3a2aec-9caa-433d-81a9-a2bb30d37274)

No issues found, closing discussion as completed
